src/collective/cover/layout.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+from collective.cover.grid import BaseGrid  # noqa
+from zope import deprecation
+
+
+deprecation.deprecated(
```

Comment:
looking [the code](https://github.com/zopefoundation/zope.deprecation/blob/master/src/zope/deprecation/deprecation.py#L171) it will monkey patch the method or module or attribute to [send a warning](https://github.com/zopefoundation/zope.deprecation/blob/master/src/zope/deprecation/deprecation.py#L145) when the method or attribute or module is called.